coralproject/cay
# Form builder enhancements

## What does this PR do?

- Adds multiple choice support on Cay Asks
- Refactor: field saving only happens when clicking "Save" explicitly, now you can close the field editor discarding all changes.

## How do I test this PR?

*Multiple choice*
- Create a form
- Add a multiple choice field
- Click on the field to edit it
- Add options to it
- Save the changes
- Click preview -> You should now see the multiple choice field with its options
- Try changing "allow multiple selections" to switch between checkboxes and radio buttons.

*Save/Discard*
- Create a form
- Add a text field
- Click on the dropped field to edit it
- Change some parameters (title, identity, required...)
- Click on the X
- Click to edit again -> field should remain as it was
- Try changing some params and saving.
- Click to edit again -> field should show you the saved parameters
